features/app-launch-handler.yml

```
@@ -0,0 +1,9 @@
+name: Launch Handler
```

Hmm, we pluralize other "handlers" for web app manifests. @captainbrosset maybe you have an opinion about this?

Also, a nit: we use title case for feature names

```suggestion
name: Launch handlers
```

(If this change is accepted, then the ID for the feature should change to match.)

---

Will wait on @captainbrosset 's feedback here.

---

We did pluralize `file_handlers` and `protocol_handlers`, but that's because the corresponding manifest members are also pluralized (because they accept arrays of handlers). The `launch_handler` member is singular in the manifest. So I think we should keep it singular.

`Launch handler`